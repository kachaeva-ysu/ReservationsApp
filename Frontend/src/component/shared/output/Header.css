.header {
    font-size: 1.8rem;
    margin-top: 1.5rem;
    margin-left: 1rem;
    margin-bottom: 1rem;
    color: #1a535c;
}

.subheader {
    composes: header;
    font-size: 1.3rem;
}