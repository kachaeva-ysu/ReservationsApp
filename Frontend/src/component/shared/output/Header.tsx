import React from 'react';
import s from "./Header.css";

type HeaderProps = {
    value: string,
    subheader?: boolean
}

const Header = ({value, subheader}: HeaderProps) => {
    return (
        <h2 className={subheader ? s.subheader : s.header}>{value}</h2>
    );
}
export default Header;
