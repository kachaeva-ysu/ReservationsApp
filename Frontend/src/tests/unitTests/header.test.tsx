import Header from "../../component/shared/output/Header";
import {expect, test} from '@jest/globals';
import renderer from 'react-test-renderer';
import React from "react";

test('Should render Header component', () => {
    const header = renderer.create(<Header value='Test header'/>).toJSON();
    expect(header).toMatchSnapshot();
});

test.only('Should render Header component 2', () => {
    const header = renderer.create(<Header value='Test header' subheader={true}/>).toJSON();
    expect(header).toMatchSnapshot();
});