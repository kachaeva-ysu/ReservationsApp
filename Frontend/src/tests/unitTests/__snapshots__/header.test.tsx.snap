// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header component 1`] = `
<h2>
  Test header
</h2>
`;

exports[`Should render Header component 2 1`] = `
<h2>
  Test header
</h2>
`;
